examples/CommonGateway/c-sensors/readme.md

```diff
@@ -1 +1,129 @@
-# To Do as per [template](../../examples_readme_blueprint.md)
\ No newline at end of file
+# Example Project: C-sensor for CommonGateway Development Kit
+
+## Goal:
+This example is for running the c-sensor in CommonGateway.
+
+## Brief:
+This example will show the enviromental measurements and dimensions.
+
+### Components Required:
+
+#### Software Components
+Please refer to the getting started [quick start](http://docs.eclipsekiso.de:1313/user-guide/quick_start.html) guide for Eclipse Kiso.
+
+#### Hardware Components
+You will need CommonGateway Board. 
+## Initial Setup
+Configure the project to be compiled for the CommonGateway Board which is located in [boards/CommonGateway](https://github.com/eclipse/kiso/tree/master/boards/CommonGateway) .
+
+## Execution Flow
+```
+cd kiso
+cmake . -Bbuild -DKISO_BOARD_NAME=CommonGateway -DKISO_APPLICATION_NAME=c-sensors -DKISO_APPLICATION_PATH=examples/CommonGateway/c-sensors 
+cmake --build build
+```
+Now as you have built the project you should have an application binary ready to run.
+For you own convenience you can use the cmake build system also to flash the binary with the external debugger.
+```
+cmake --build build --target flash
+```
+
+## Code Walkthrough
+The essence of this example can be seen in the appPeriodicTest function, which is taking the data from  SensorEnvironment_Read and make some calculations to give the output understandable and take the data from the SensorAccelerometer_Read and show dimensions and if everything goes correct, the green led will flash. Otherwise the red led will flash .
```

Comment:
```suggestion
The main program loop can be seen in `appPeriodicTest()`. It takes data from  `SensorEnvironment_Read()` and `SensorAccelerometer_Read()`, performs some conversions and corrections based on static calibration and finally presents the data via debug log. If everything went OK, the green LED will flash. In case of any errors the red LED will flash.
```